ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.15.yaml

```diff
@@ -1,121 +0,0 @@
-base_images:
-  ansible:
-    name: "4.15"
-    namespace: ocp
-    tag: ansible
-  cli:
-    name: "4.15"
-    namespace: ocp
-    tag: cli
-  dev-scripts:
-    name: test
-    namespace: ocp-kni
-    tag: dev-scripts
-  openstack-installer:
-    name: "4.15"
-    namespace: ocp
-    tag: openstack-installer
-  tests-private-postupg:
-    name: tests-private
-    namespace: ci
-    tag: "4.15"
-  tests-private-preupg:
-    name: tests-private
-    namespace: ci
-    tag: "4.15"
-  tools:
-    name: "4.15"
-    namespace: ocp
-    tag: tools
-  upi-installer:
-    name: "4.15"
-    namespace: ocp
-    tag: upi-installer
-  verification-tests:
-    name: verification-tests
-    namespace: ci
-    tag: latest
-build_root:
-  image_stream_tag:
-    name: release
-    namespace: openshift
-    tag: golang-1.18
-releases:
-  latest:
-    prerelease:
-      architecture: amd64
-      product: ocp
-      relative: 1
-      version_bounds:
-        lower: 4.15.0-0
-        upper: 4.16.0-0
-  target:
-    prerelease:
-      architecture: amd64
-      product: ocp
-      version_bounds:
-        lower: 4.15.0-0
-        upper: 4.16.0-0
-resources:
-  '*':
-    requests:
-      cpu: 100m
-      memory: 200Mi
-tests:
-- as: aws-ipi-disc-priv-tp-f60
-  cron: 24 17 12 2,4,6,8,10,12 *
-  steps:
-    cluster_profile: aws-qe
-    env:
-      BASE_DOMAIN: qe.devcluster.openshift.com
-      ENABLE_OTA_TEST: OCP-69968
```

Comment:
@jiajliu can we delete the job with the OTA cases here ?

Reply:
yes, we can